app/src/main/res/layout/fragment_closet_menu.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/fragment_closet_menu_constraint_layout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:layout_gravity="right"
+    android:background="@drawable/closet_menu_fragment_background"
+    android:focusable="true"
+    android:minWidth="150dp"
+    android:orientation="vertical">
+
+
+    <Button
```

Comment:
Not sure if you remember the message I sent you a little while ago on WA. If you change these to imageviews and make them clickable and focusable, then they will stretch the entire width of their parent without any borders. I think it will look a lot cleaner